troi/content_resolver/database.py

```diff
@@ -22,7 +22,7 @@
 
 from troi.content_resolver.utils import existing_dirs
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("troi_db_scan")
```

Comment:
I think in general its better to retain the original name of the file, why do you want to modify the logger name here?

Reply:
so its a known and documented name, so that LB Local can capture the logger and its output for when LB local scans collections via troi.